plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/style.scss

```diff
@@ -28,6 +27,7 @@
 	&[aria-checked="true"] {
 		background-color: $black;
 		color: $white;
+		border-color: $black;
```

Comment:
> In this PR, I’ve added `border-color: $black;` for the selected pills. This change was necessary because, the border was being set to white due to the use of `currentColor` for the border-color.

It makes sense, thanks for noticing that!